include/flexflow/model.h

```diff
@@ -267,6 +268,42 @@ class Replicate;
 class FusedParallelOp;
 class ParallelOpInfo;
 
+// TODO: Move to an appropriate place
+/*
```

Comment:
Might be good to convert this to doxygen format at some point. Doesn't have to be this PR.